.changeset/early-students-tie.md

```diff
@@ -0,0 +1,43 @@
+---
+"@turnkey/eip-1193-provider": minor
+"@turnkey/sdk-browser": major
+"@turnkey/sdk-server": minor
+"@turnkey/sdk-react": major
+---
+
+## Major Package Updates
+
+### @turnkey/sdk-browser
+
+- create abstract `TurnkeyBaseClient` class which extends `TurnkeySDKClientBase`
+  - `TurnkeyBrowserClient`, `TurnkeyIframeClient`, `TurnkeyPasskeyClient`, and `TurnkeyWalletClient` all extend `TurnkeyBaseClient`
+- TurnkeyBrowserClient
+  - Session Management
+    - refreshSession - attempts to refresh an existing session and will extend the session expiry using the expirationSeconds parameter
```

Comment:
optional nit: we can markdown-ify these functions and code-specific params like `expirationSeconds`, e.g. 
```suggestion
    - `refreshSession` - attempts to refresh an existing session and will extend the session expiry using the `expirationSeconds` parameter
```